### PR TITLE
[FIX] runbot: addapt 18.0

### PR DESCRIPTION
**Pull request by Xavier-Do**

### CODE REVIEW

File: runbot/models/branch.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def _compute_branch_url(self):
                 if branch.is_pr:
                     branch.branch_url = "https://%s/pull/%s" % (branch.remote_id.base_url, branch.name)
                 else:
-                    branch.branch_url = "https://%s/tree/%s" % (branch.remote_id.base_url, branch.name)
+                    branch.branch_url = "https://%s/list/%s" % (branch.remote_id.base_url, branch.name)
```
Comment (Williambraecky) [inline runbot/models/branch.py]:
```suggestion
                    branch.branch_url = "https://%s/tree/%s" % (branch.remote_id.base_url, branch.name)
```

File: runbot/templates/badge.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <data>
     <template id="runbot.badge_default">&lt;?xml version="1.0"?&gt;
       <svg t-attf-xmlns="http://www.w3.org/2000/svg" t-attf-width="{{left.width + right.width}}" height="18">
-        <!-- from https://github.com/badges/shields/tree/master/templates -->
+        <!-- from https://github.com/badges/shields/list/master/templates -->
```
Comment (Williambraecky) [inline runbot/templates/badge.xml]: The link seems completely dead, it seems they don't provide 'templates' anymore?
I didn't check for too long though, perhaps we could change it to: 
```suggestion
        <!-- inspired by https://github.com/badges/shields/ -->
```

File: runbot/tests/test_branch.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 class TestBranch(RunbotCase):
 
     def test_base_fields(self):
-        self.assertEqual(self.branch_server.branch_url, 'https://example.com/base/server/tree/master')
+        self.assertEqual(self.branch_server.branch_url, 'https://example.com/base/server/list/master')
```
Comment (Williambraecky) [inline runbot/tests/test_branch.py]:
```suggestion
        self.assertEqual(self.branch_server.branch_url, 'https://example.com/base/server/tree/master')
```

File: runbot/templates/badge.xml (hunk 2 above)
Comment (Xavier-Do) [reply runbot/templates/badge.xml]: We may check that, but for now I would just revert the invalid changes made by upgrade_code

```suggestion
        <!-- from https://github.com/badges/shields/list/master/templates -->
        <!-- from https://github.com/badges/shields/tree/master/templates -->
```

**Review by Williambraecky — APPROVED** [review]

**Pull request merged**